tools/depgraph/depgraph.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+
+# UPPERCASE THE FIRST LETTER OF A WORD
+function uppercasefirst ()
+{
+  word=$1
+  first=${word:0:1}
+  firstupper=${first^^}
+  wordwithoutfirst=${word:1}
+  echo "${firstupper}${wordwithoutfirst}"
+}
+
+errordirectory="error : non-existing directory"
+workingdir=$(pwd)
+scriptdir=$(dirname "${BASH_SOURCE[0]}")
+lambdapidir=$(cd ${scriptdir}/../.. || { echo $errordirectory ; exit 1; } ; pwd)
+graph=graph
+graphdot=$workingdir/$graph.dot
+graphpng=$workingdir/$graph.png
+
+# CHOOSE THE INTERESTING_MODULES MODULES
+interesting_modules=( compile handle infer proof sr tactics typing unif )
```

Comment:
Perhaps we could specify the modules on command line? The simplest way seems to use `"$@"` as `interesting_modules` (perhaps you'll have to use `modules=( "$@" )` to convert to a proper array).